lib/charms/traefik_k8s/v0/ingress_per_leader.py

```diff
@@ -5,15 +5,16 @@
 
 This library wraps relation endpoints using the `ingress` interface
 and provides a Python API for both requesting and providing per-application
-ingress, with load-balancing occurring across all units.
+ingress, without load balancing across Traefik units: all traffic is routed
+through the leader instead.
```

Comment:
Should we further warn the reader?
```suggestion
through the leader instead.
This is not a typical ingress mode and is intended only for special cases.
```